OODebug/src/debugger/JavaDebugger.cpp

```diff
@@ -69,6 +69,22 @@ struct VariableObserver {
 		QList<Probes::ValueCalculator> valueCalculators_;
 };
 
+struct UserVisibleBreakpoint {
+		qint64 breakpointId_;
```

Comment:
It might be nicer to define a custom breakpoint type, even if it's just `using BreakpointId = int64`.